Gruntfile.js

```diff
@@ -50,17 +50,30 @@ module.exports = function(grunt) {
         },
       },
     },
-    'qunit': {
-      files: ['test/unit.html', 'test/integration.html'],
+    qunit: {
```

Comment:
NIT: We should have this be consistent and either wrap attributes in single quotes or not at all

Reply:
Fixed